package.json

```
@@ -1,14 +1,15 @@
 {
   "name": "fhir-bundle-calculator",
-  "version": "2.0.0",
-  "description": "CLI for executing CQL calculation for a generated Synthea Patient Bundle",
+  "version": "3.0.0",
+  "description": "A CLI for outputting population statistics and MeasureReports for FHIR patients for an eCQM using the $evaluate-measure operation",
   "repository": "git@github.com:projecttacoma/fhir-bundle-calculator.git",
   "author": "Matthew Gramigna <mgramigna@mitre.org>",
   "main": "src/cli.js",
   "scripts": {
-    "lint": "eslint ./**/*.js",
-    "lint-fix": "eslint ./**/*.js --fix",
-    "test": "NO_PROXY=localhost jest"
+    "lint": "eslint \"./**/*.js\"",
+    "lint-fix": "eslint \"./**/*.js\" --fix",
+    "test": "NO_PROXY=localhost jest",
+    "test:watch": "NO_PROXY=localhost jest --watchAll"
```

```suggestion
    "test": "jest",
    "test:watch": "jest --watchAll"
```

Shouldn't need the `NO_PROXY` var any more, I don't think?